dev/tests/api-functional/testsuite/Magento/GraphQl/CustomerDownloadableProduct/CustomerDownloadableProductTest.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\CustomerDownloadableProduct;
+
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+use Magento\TestFramework\ObjectManager;
+use Magento\TestFramework\TestCase\GraphQl\ResponseContainsErrorsException;
+use Magento\Customer\Api\CustomerRepositoryInterface;
+use Magento\Sales\Api\OrderRepositoryInterface;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+
+class CustomerDownloadableProductTest extends GraphQlAbstract
+{
+
+    /**
+     * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+     * @magentoApiDataFixture Magento/Customer/_files/customer.php
+     * @magentoApiDataFixture Magento/Downloadable/_files/product_downloadable.php
+     * @magentoApiDataFixture Magento/Downloadable/_files/order_with_downloadable_product.php
+     */
+    public function testGetCustomerDownloadableProducts()
```

Comment:
This test can be simplified. Please have a look at `\Magento\GraphQl\Downloadable\CustomerDownloadableProductsTest::testCustomerDownloadableProducts`.